scripts/overton/duplicate_filter_script.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import typer
+
+def main(overton_excel_path : str, duplicate_only_path : str, no_duplicates_path : str, duplicate_report_path : str):
+
+    data = pd.read_excel(overton_excel_path)
+    df   = pd.DataFrame(data)
+    df   = df[['Title', 'Published', 'URL', 'PDF URL']]
+
+    df['Duplicate'] = [False] * len(df)
+
+    nr_rule_1 = 0
+    nr_rule_2 = 0
+    nr_rule_3 = 0
+    nr_rule_4 = 0
+    pairs     = []    
+    for i in range(len(df)) :

```

Comment:
jak się iterujemy po df to lepiej for index, row in df.iterrows()

Reply:
wtedy mamy dostęp do całego wiersza i indeksu jednocześnie

Reply:
Spoko, ale potem w kolejnej części kodu iteruje od j które zaczyna się po i, więc tam już iterrows niezbyt jest jak użyć i to ostatecznie nie zmienia wiele